api/controllers/v1/jobs.ts

```diff
@@ -222,3 +228,33 @@ async function NotifyBuildProgress(jobId: string): Promise<any> {
     statusCode: 200,
   };
 }
+
+function getMongoClient(config: IConfig): mongodb.MongoClient {
+  const url = `mongodb+srv://${config.get('username')}:${config.get('password')}@${config.get(
+    'host'
+  )}/?retryWrites=true&w=majority`;
+  return new mongodb.MongoClient(url);
+}
+
+async function SubmitArchiveJob(jobId: string) {
```

Comment:
Is there a reason for why we can't use the client defined on line 16 here, passed in as an arg?